app/src/main/java/com/github/h3lp3rs/h3lp/messaging/Conversation.kt

```diff
@@ -16,26 +22,153 @@ class Conversation(
     private val currentMessenger: Messenger
 ) {
     private val database = databaseOf(MESSAGES)
+    private var publicKey: PublicKey? = null
+    private val allMessages: HashMap<String, String> = HashMap()
```

Comment:
Beware, this object gets lost when the activity closes. Isn't a storage more preferable?